app/scripts/models/notifications.js

```diff
@@ -15,7 +15,9 @@ define([
   var EVENTS = {
     DELETE: 'fxaccounts:delete',
     LOGOUT: 'fxaccounts:logout',
-    PROFILE_CHANGE: 'profile:change'
+    PROFILE_CHANGE: 'profile:change',
+    SIGNED_IN: 'fxaccounts:signedIn',
+    SIGNED_OUT: 'fxaccounts:signedOut'
```

Comment:
Actually that's why I deliberately picked a different message. That logout message is already being sent and I wasn't sure that I could rely on it / the timing would be right. What is that message used for, who sends it, when is it sent etc?

I also specifically wanted my event name to be in the past tense, because it is a signal that something has already happened rather than a command.